src/app/payment/paymentMethod/StripeUPEPaymentMethod.tsx

```diff
@@ -87,9 +87,8 @@ const StripeUPEPaymentMethod: FunctionComponent<StripePaymentMethodProps & WithI
             hideContentWhenSignedOut
             initializePayment={ initializeStripePayment }
             method={ method }
-            renderCustomPaymentForm={ renderCustomPaymentForm }
-            shouldRenderCustomInstrument={ true }
         />
+        { renderOptimizedCheckoutStyles() }
```

Comment:
Moving `renderOptimizedCheckoutStyles` out creates a new use case for `HostedWidgetPaymentMethod`. Can we avoid this by improving `HostedWidgetPaymentMethod` component?